pkg/virt-operator/resource/generate/rbac/cluster.go

```diff
@@ -162,6 +162,8 @@ func newAdminClusterRole() *rbacv1.ClusterRole {
 					"virtualmachineinstances/unfreeze",
 					"virtualmachineinstances/softreboot",
 					"virtualmachineinstances/portforward",
+					"virtualmachineinstances/addinterface",
```

Comment:
Thanks.
The commit message says - 
> The `admin`, and `edit` cluster roles are updated, granting RBAC access
> to the `addinterface` VMI subresource.

It should be VMI and VM subresource (although I have no idea in what commit the VM resource RBAC rules were added, it should be here).

Please also mention it gives the virt-controller also can update the VM subresource. It is needs since when the VM subresource the virt-controller is responsible to update the VMI subresource.